client/src/pages/JoinPage/style.scss

```diff
@@ -1,5 +1,38 @@
+@import "@/constants/style";
+
 .visualizer {
     height: 10px;
     background: #FDEEC8;
     transition: width 0.2s;
 }
+
+.password-wrapper {
+  display: flex;
+  align-items: center;
+  margin-top: 16px;
+
+  .password-toggle {
+    border-radius: 100%;
+    width: 2.5rem;
+    height: 2.5rem;
+    margin-right: 4px;
+    background-color: $gray;
+
+    &.locked-style {
+      background-color: $orange;
+    }
+  }
+  
+  div {
+    flex-grow: 1;
+
+    .password-input.disabled {
+      pointer-events: none;
+      background-color: $disabled-background-color;
+
+      &::placeholder {
```

Comment:
Is this applied to `.password-input.disabled` ? or just `.password-input`

Reply:
It's just applied to elements with both `.password-input` and `.disabled` so just `.password-input.disabled`